backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/ObservationHandler.java

```diff
@@ -86,7 +86,7 @@ public ObservationHandler(Executor executor, FutureJdbi jdbi) {
             handle ->
                 handle
                     .createQuery(
-                        "select k.kv_key _key, k.kv_value _value, k.value_type _type, o.epoch_number epoch, o.experiment_run_id run_id "
+                        "select k.kv_key _key, k.kv_value _value, k.value_type _type, o.epoch_number epoch, o.experiment_run_id run_id, o.timestamp "
```

Comment:
`timestamp` field is missing in Observation while we are returning it which i have fixed.